django/django.go

```diff
@@ -23,6 +23,8 @@ type Engine struct {
 	forwardPath bool
 	// templates
 	Templates map[string]*pongo2.Template
+	// set auto escape globally
+	AutoEscape bool
```

Comment:
I think this one can be unexported property since we have `SetAutoEscape`

Reply:
Done